man/visualSubsetRecode.Rd

```diff
@@ -0,0 +1,60 @@
+\name{visualSubsetRecode}
+\alias{visualSubsetRecode}
+\title{Inspect data subsets visually and recode instantaneously}
+\description{
+This function allows manual inspection and manual recoding of subsets of the data.}
+\usage{visualSubsetRecode(dat, subsetInfo, ID, toRecodeVal="mci", useGroups=NULL)}
+\arguments{
+  \item{dat}{A data frame}
+  \item{subsetInfo}{A data frame. Colnames have to be: "datCols" and whatever is specified in ID and useGroups.}
+  \item{ID}{A character vector of length 1 indicating the name of the case identifier variable in \code{dat}. Default is \code{ID}.}
+  	\item{toRecodeVal}{Optional: A scalar indicating recode string. }
+  	\item{useGroups}{Optional: A character string indicating the name of the group identifier column in subsetInfo. Used for group level subsetting. }
```

Comment:
It should be added according to which criteria the data are optionally recoded (and why).

Reply:
Thanks, that is much easier to understand. Following on from the example from the DESCRIPTION, I would add the following for \item{subsetInfo}: "The subsetInfo data frame would first have to be created by the user from the test session logs. If, for example, the log shows that audio X in test group Y was possibly broken, all person IDs of this test group Y should be entered in subsetInfo in [ID] column and all variables belonging to this audio X in [datCols]. In this way, the responses for this audio and this test group can be checked, flagged and recoded to mci if necessary.

Reply:
I'll update this as soon as the createSubsetInfo()-function is ready. Once that's done, I can include it in the description. Does that work for you?

If yes, can I merge branches now and then open a new pull request whenever the createSubsetInfo() is ready for code review?

Reply:
One thing that is still unclear to me: The function offers the options yes, no, flag, go back and reset to original values. Some of these options could be explained in more detail. The function returns a list with two data.frames. Ideally, the list could be named as the input arguments are: “dat” and "subsetInfo". It could then be added that option “yes” changes something in “dat”, but option “flag” only in “subsetInfo”. I also don't quite understand the “reset” option. Does this mean that you can call the function several times in succession and, for example, undo recodings?

Reply:
Please add a line break after \dontrun{, otherwise the first line of the command will be commented out in the rendered help page.

Reply:
> One thing that is still unclear to me: The function offers the options yes, no, flag, go back and reset to original values. Some of these options could be explained in more detail. The function returns a list with two data.frames. Ideally, the list could be named as the input arguments are: “dat” and "subsetInfo". It could then be added that option “yes” changes something in “dat”, but option “flag” only in “subsetInfo”. I also don't quite understand the “reset” option. Does this mean that you can call the function several times in succession and, for example, undo recodings?

Yes, what a good idea! I will add the labels to the documentaion. And, yes, it means that once I've set something to "mci" I can go back and reset it to the original values...

Reply:
Btw, I moved this conversation now to this issue, as this will be the place where the next changes will take place: https://github.com/sachseka/eatPrep/issues/64